### ISSUE TITLE
org.prorefactor.refactor.RefactorException : unexpected token: SERIALIZE-HIDDEN

### ISSUE THREAD

**Issue by jcaillon** [oehive/proparse#7]
Hello,

First of all, thank you for this project :)

I've encountered a bug in the proparse, there seems to be an unknown keyword `SERIALIZE-HIDDEN` that triggers an exception.

Here is a code sample to reproduce the bug :

```
DEFINE TEMP-TABLE table1 NO-UNDO
    FIELD field1 AS CHARACTER
    INDEX idx field1
    .

DEFINE DATASET dataset1
    SERIALIZE-HIDDEN
    FOR table1.
```

Thank you for considering this issue

**Comment by mikefechner** [comment]
This is fixed in https://github.com/consultingwerk/proparse

**Comment by jcaillon** [comment]
Oh, I saw your message a bit too late. I'll check this fork and https://github.com/Riverside-Software/proparse/network which also seems ahead
Thank you

**Comment by mikefechner** [comment]
It’s not a true fork …. We started off the original SVN repo separately. But the code is very similar.

Von: Julien Caillon [mailto:notifications@github.com]
Gesendet: Freitag, 14. Oktober 2016 15:22
An: oehive/proparse proparse@noreply.github.com
Cc: Mike Fechner mike.fechner@consultingwerk.de; Comment comment@noreply.github.com
Betreff: Re: [oehive/proparse] org.prorefactor.refactor.RefactorException : unexpected token: SERIALIZE-HIDDEN (#7)

Oh, I saw your message a bit too late. I'll check this fork and https://github.com/Riverside-Software/proparse/network which also seems ahead
Thank you
